src/connect/hooks/tree-view/usePathContent/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './usePathContent';
+export { default } from './usePathContent';
```

Comment:
what's up with exporting the default exports like this? you could just export the hook itself and not have a default export. that would achieve the same result as doing this without needing the extra export